applications/callflow/src/module/cf_record_call.erl

```diff
@@ -29,7 +29,9 @@ handle(Data0, Call) ->
 
 -spec handle(kz_json:object(), kapps_call:call(), kz_term:ne_binary()) -> kapps_call:call().
 handle(Data, Call, <<"start">>) ->
-    cf_exe:update_call(kapps_call:start_recording(Data, Call));
+    Call1 = kapps_call:kvs_store('recording_follow_transfer', 'true', Call),
```

Comment:
can we get the desired behavior from Data object (we can still have it default to true) ?

Reply:
That's a good point, I'll add that